.github/assets/test-upgrade-manifest.yml

```diff
@@ -1,110 +0,0 @@
-# Namespace
```

Comment:
I added it in this[ PR to validate the upgrade](https://github.com/kedacore/keda/pull/2450), but finally we are not doing that because it was more complex than expected